sdk/ai/ai-inference-rest/samples/v1-beta/javascript/README.md

```diff
@@ -11,14 +11,15 @@ urlFragment: ai-inference-javascript-beta
 
 These sample programs show how to use the JavaScript client libraries for Azure AI Inference in some common scenarios.
 
-| **File Name**                                     | **Description**                       |
-| ------------------------------------------------- | ------------------------------------- |
-| [chatCompletions.js][chatcompletions]             | get chat completions.                 |
-| [embeddings.js][embeddings]                       | get embeddings.                       |
-| [getModelInfo.js][getmodelinfo]                   | get model info.                       |
-| [imageFileCompletions.js][imagefilecompletions]   | get chat completions with image file. |
-| [streamChatCompletions.js][streamchatcompletions] | list chat completions.                |
-| [toolCall.js][toolcall]                           | get chat completions with functions.  |
+| **File Name**                                     | **Description**                                |
+| ------------------------------------------------- | ---------------------------------------------- |
+| [chatCompletions.js][chatcompletions]             | get chat completions.                          |
+| [embeddings.js][embeddings]                       | get embeddings.                                |
+| [getModelInfo.js][getmodelinfo]                   | get model info.                                |
+| [imageFileCompletions.js][imagefilecompletions]   | get chat completions with image file.          |
+| [streamChatCompletions.js][streamchatcompletions] | list chat completions.                         |
+| [streamingToolCall.js][streamingtoolcall]         | use tool call with streaming chat completions. |
+| [toolCall.js][toolcall]                           | get chat completions with functions.           |
```

Comment:
Start all description with capital letter.

Also, in the description for tool samples, for one sample you say "use tool call" and the other you say "get chat completions with functions".  I would be more consistent in terminology, to indicate that the samples are similar but just differ on whether streaming was used or not. So perhaps:
"Get chat completions with streaming and function call"
"Get chat completions with function call"